.pre-commit-config.yaml

```diff
@@ -3,12 +3,17 @@ repos:
     rev: v4.0.1
     hooks:
       - id: check-docstring-first
+        language_version: python3
```

Comment:
Could you revert these? Not sure where this is coming from.

Reply:
These were added by me to avoid getting errors as the one attached below whenever I made a commit
`An unexpected error has occurred: CalledProcessError: command: ('/home/dbhasin1/.conda/envs/vision-cpu/bin/python3.1', '-mvirtualenv', '/home/dbhasin1/.cache/pre-commit/repox5vbpbp2/py_env-python3.1', '-p', 'python3.1')
return code: 1
expected return code: 0
stdout:
    RuntimeError: failed to find interpreter for Builtin discover of python_spec='python3.1'
    
stderr: (none)`